HelpfulItem.h

```diff
@@ -10,6 +10,5 @@
 struct HelpfulItem : Item
```

Comment:
You are including a bunch of header files above this line.
Don't include what you don't use
- [ ] revise